docs/guides/sqlite3_db_with_concord.md

```diff
@@ -0,0 +1,259 @@
+# Other topics
+
+## Using a DB with concord
+
+Sometimes, when you are developing a command, you may wish to save some data, that will be not deleted across the restarts from your bot, this is where a database comes in. It can save data, and, don't worry, it will not be deleted if you restart your bot.
+
+In this guide, we will be teaching you how to use sqlite3, a local database, that it's incredibly fast, easy to use and debug.
```

Comment:
"In this guide, we will be teaching you how to use sqlite3, a local database. It's incredibly fast, easy to use, and simple to debug."

Reply:
Added, thanks